packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -31,16 +31,20 @@ import { WorkspaceAdapter } from '../../../services/workspace-adapter';
 import {
   DevWorkspaceClient,
   DEVWORKSPACE_NEXT_START_ANNOTATION,
+  EditorDefinition,
   IStatusUpdate,
 } from '../../../services/workspace-client/devworkspace/devWorkspaceClient';
 import { selectRunningWorkspacesLimit } from '../../ClusterConfig/selectors';
 import { createObject } from '../../helpers';
 import { selectDefaultNamespace } from '../../InfrastructureNamespaces/selectors';
-import * as DwPluginsStore from '../../Plugins/devWorkspacePlugins';
-import { selectDwEditorsPluginsList } from '../../Plugins/devWorkspacePlugins/selectors';
+import * as PluginsStore from '../../Plugins/devWorkspacePlugins';
+import {
+  selectDefaultEditor,
```

Comment:
Do we really need both  'selectDefaultEditor' and 'selectDefaultEditorCR'?

Reply:
You're right, now I see `selectDefaultEditorCR` is redundant.